viam-cartographer.go

```diff
@@ -133,7 +133,8 @@ func New(
 		return nil, err
 	}
 
-	cancelCtx, cancelFunc := context.WithCancel(ctx)
+	// Need to pass in a long-lived context because ctx is short-lived
+	cancelCtx, cancelFunc := context.WithCancel(context.Background())
```

Comment:
Can we talk about this change in person? I know this was one of the issues you and Zack were looking into, but I don't fully understand it.

Reply:
Yes, we should talk about it with Nick as well. You might want to come to our meeting in C8 at 15:00 for the first 10 minutes so we can talk about this.

Reply:
Sounds great, thank you!

Reply:
I changed this to context.Background() to be in line with orbslam based on recent learnings: https://github.com/viamrobotics/viam-orb-slam3/pull/29/files#diff-c70dd5ed993c1b53cd893d5a38fa4665c771456384a369b2a67b7a55ddb9fffeR341

Reply:
Thank you! Resolving